src/utils/types.ts

```diff
@@ -98,7 +98,7 @@ type ResponseFull = {
   text: string | null | undefined
   optionIndex?: number
   reference?: any // Not yet decided how to represent
-  isValid: boolean | null
+  isValid?: boolean | null
```

Comment:
Can this be null?

Reply:
I think so. All Response parameters start off as NULL when the Application is created, right?